include/constants/battle_ai.h

```diff
@@ -34,20 +34,20 @@
 #define AI_FLAG_PREFER_BATON_PASS     (1 << 6)
 #define AI_FLAG_DOUBLE_BATTLE         (1 << 7)   // removed, split between AI_FLAG_CHECK_BAD_MOVE & AI_FLAG_CHECK_GOOD_MOVE
```

Comment:
I feel like we should perhaps replace removed flags at some points rather than just adding more at the end

Reply:
I agree but this one isn't actually removed. The flag is still used in every double battle. I don't know why the comment was added.

Reply:
Used in every battle in trainers.h or actually checked in the game logic?

Reply:
Ye, when it is a double battle it is added as a trainer flag.

Reply:
If it doesn't have any functionality we should probably just deprecate it then, but that's stuff for another PR

Reply:
From a first look it does have some functionality but it looks like a lot of it are duplicates. I can look into removing it.